scan/schema.go

```diff
@@ -298,7 +298,8 @@ func (scp *schemaParser) parseDecl(definitions map[string]spec.Schema, decl *sch
 			}
 		}
 	default:
-		return fmt.Errorf("this is a bug, missing parser for a %T: %+v", tpe, tpe)
+		fmt.Printf("WARNING: Missing parser for a %T, skipping model: %s\n", tpe, decl.Name)
```

Comment:
can you make this use `log.Printf` or `fmt.Sprintf(os.Stderr, ...)`
the other things that get printed are also using log.